ara/ui/templates/index.html

```diff
@@ -134,8 +134,20 @@
             </td>
             <td class="text-nowrap">{{ playbook.started | format_date }}</td>
             <td class="text-nowrap">{{ playbook.duration | format_duration }}</td>
-            <td class="text-center">{{ playbook.items.hosts }}</td>
-            <td class="text-center">{{ playbook.items.tasks }}</td>
+            <td class="text-center">
+              {% if not static_generation %}
+                <a href="{% url 'ui:host_index' %}?playbook={{ playbook.id }}" title="Search for hosts in this playbook">{{ playbook.items.hosts }}</a>
```

Comment:
Good catch on the missing ``latest=false``, thanks !

I'm not entirely convinced of the worth of this particular link (versus tasks which has value IMO) but it's "free" and I suppose since I was adding a link for tasks, I might as well add a link for hosts too.

I figure that it's not necessarily because I don't have a use case for this specific link that it doesn't mean it can't be useful to someone else, though.

Reply:
yes it is "free" but on the other side it increases complexity. Complexity for the users to navigate on the side (imo it can be confusing if there are many of links on one page), but also more complex for development. 
Afaik we don't have any tests which checks if links are still working in the way they are assumed to. Thus there is always the dangerour that links will be broken after ui updates because the developer didn't see that there also links pointing to a certain functionality, especially if the links are on other pages.